test/include/seqan3/test/performance/units.hpp

```diff
@@ -36,11 +37,52 @@ inline benchmark::Counter bytes_per_second(size_t bytes)
 
 //!\brief Calculates the number of cell updates for given sequences for a specific alignment config.
 template <typename sequences_range_t>
-inline size_t pairwise_cell_updates(sequences_range_t const & sequences_range, auto && /*align_cfg*/)
+inline size_t pairwise_cell_updates(sequences_range_t const & sequences_range, [[maybe_unused]] auto && align_cfg)
 {
+    auto count_cells = [&] (auto && seq1, auto && seq2)
+    {
+        size_t const max_col_count = std::ranges::size(seq1) + 1;
+        size_t const max_col_size = std::ranges::size(seq2) + 1;
+
+        if constexpr (align_cfg.template exists<seqan3::align_cfg::band_fixed_size>())
+        {
+            using std::get;
+            auto band_cfg = get<seqan3::align_cfg::band_fixed_size>(align_cfg);
+
+            int32_t lower_diagonal = band_cfg.lower_diagonal.get();
+            int32_t upper_diagonal = band_cfg.upper_diagonal.get();
+            size_t band_size = upper_diagonal - lower_diagonal + 1;
+
```

Comment:
I know I spent too much time on this, but how about this slightly shorter version:

```suggestion
        size_t const columns = std::ranges::size(seq1) + 1;
        size_t const rows = std::ranges::size(seq2) + 1;

        // ...

            int32_t lower_diagonal = band_cfg.lower_diagonal.get();
            int32_t upper_diagonal = band_cfg.upper_diagonal.get();

            for (int32_t column_id = 0; column_id < columns; ++column_id)
            {
                // the position of the upper-row (inclusive)
                int32_t upper_row_id = std::clamp<int32_t>(column_id - upper_diagonal, 0, rows);
                // the position of the lower-row (exclusive)
                int32_t lower_row_id = std::clamp<int32_t>(column_id - lower_diagonal + 1, 0, rows);

                matrix_cells += lower_row_id - upper_row_id;
            }
```

Reply:
I don't care. As long as it works :)

Reply:
What actually is replaced here?

Reply:
Ah ok, now I got it.